README.md

```diff
@@ -1,4 +1,4 @@
-# HTTP Client v0.4.1
+# HTTP Client v0.4.2
```

Comment:
We should probably bump minor version instead since we extend functionality

Reply:
But isn't it functionality that more closer to th2 side? I dont think that we need to update minor version every time while we are implementing th2 logic that should have been from definition of connects

Reply:
I don't think it matters if it's closer to th2 or not and as simple as: we've added somthing new - we bump minor version